File: app/src/main/java/com/xyz/test/DpUtils.java
package com.xyz.test;

import android.content.Context;
import android.util.TypedValue;

public class DpUtils {

    public static int dp2px(Context context, float dpVal) {
        return (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dpVal,
                context.getResources().getDisplayMetrics());
    }
}

File: app/src/main/java/com/xyz/test/DpUtils.kt
package com.xyz.test

import android.content.Context
import android.util.TypedValue

object DpUtils {

    fun dp2px(context: Context, dpVal: Float): Float {
        return TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dpVal,
                context.resources.displayMetrics)
    }
}
